README.md

```diff
@@ -2,26 +2,49 @@
 
 ![Screenshot from 2023-03-18 15-29-05](https://user-images.githubusercontent.com/8228060/226112255-fe2d4cdc-193e-4c23-8a36-3d8f60baaf03.png)
 
+## What is QuickPizza? 🍕🍕🍕
+
+`QuickPizza` is a web application, used for demonstrations and workshops, that generates new and exciting pizza combinations! 
+
+The app is built using [SvelteKit](https://kit.svelte.dev/) for the frontend and [Go](https://go.dev/) for the backend.
+
+It also demonstrates the basic and advanced functionalities of k6, ranging from a basic load test to using different modules and extensions.
```

Comment:
NIT: The app doesn't demonstrate the functionalities of k6. The tests do, and we use the app to exemplify. 

I would rephrase this, but not sure how.